### ISSUE TITLE
Bug: defined unit conversations don't show up in recipes

### ISSUE THREAD

**Issue by Thatkookooguy** [grocy/grocy#1251]
**TL;DR:**
- grocy should allow me to convert to any measurements that grams can be converted to because of the **MSG's QU conversions product override** and **gram's Default conversions** (or **tablespoon's Default conversions**).
- shouldn't the definition of conversion from 1 unit to another define automatically the reverse conversion?
---
I defined the following product:

### MSG
**stock unit:** `piece`
**purchase unit:** `piece`
**QU conversions product override:** `1 Piece is the same as 500 gram`

---
I defined the following **Quantity units**: started out with only the conventions for `teaspoon -> grams` and `tablespoon -> grams` but tried to add the other way as well to see if that helps:

### tablespoon
**Default conversions:**
- `1 tablespoon is the same as 14.3 grams`
- `1 tablespoon is the same as 3.0 teaspoons`

### teaspoon
**Default conversions:**
- `1 teaspoon is the same as 4.766 grams`

### gram
**Default conversions:**
- `1 gram is the same as 0.059 tablespoons`
- `1 gram is the same as 0.18 teaspoons`

---
I wanted to add a recipe and tell it that **I need 1.5 tablespoons of MSG**. But the only measurement options I got was **pieces and grams**.

**Expectation:** grocy should allow me to convert to any measurements that grams can be converted to because of the **MSG's QU conversions product override** and **gram's Default conversions** (or **tablespoon's Default conversions**).

Also, shouldn't the definition of conversion from 1 unit to another define automatically the reverse conversion? sounds reasonable that if I'm converting grams to tablespoons, I'll probably convert the other way around. Also, defining tablespoons to grams conversion is easier than grams to tablespoons.

---
Not sure if this is a bug or a feature. sounds to me like it should happen by default so I reported as a bug.

## Screenshots
![image](https://user-images.githubusercontent.com/10427304/103483553-bf2fc200-4df0-11eb-8ee6-b5923b190177.png)
![image](https://user-images.githubusercontent.com/10427304/103483564-cf47a180-4df0-11eb-85c8-33e44cec84a3.png)
![image](https://user-images.githubusercontent.com/10427304/103483576-df5f8100-4df0-11eb-8675-46972ffc0764.png)
![image](https://user-images.githubusercontent.com/10427304/103483585-ee463380-4df0-11eb-8195-e27b86430d55.png)

**Comment by berrnd** [comment]
Duplicate of #1217, already fixed, not yet released.

Please always try to reproduce bugs on the [pre-release demo](https://demo-prerelease.grocy.info) first, thanks.

> shouldn't the definition of conversion from 1 unit to another define automatically the reverse conversion?

Is already the case, when create a conversion there is this checkbox, checked by default:

![grafik](https://user-images.githubusercontent.com/2161815/103483772-ddde8a80-4de9-11eb-8253-8bf0e1eafac5.png)

**Comment by Thatkookooguy** [comment]
Thanks for the quick reply, @berrnd ! great product!
I hope I'll start contributing some code soon :-)
